index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bee Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="nav">
+                <img src="./ImageP1/hamburger.png" alt="hamburger">
+            </nav>
+        </header>
```

Comment:
- [ ] [OPTIONAL] Great job 👍 adding `header` tag as requested by previous reviewer however I would suggest using `header` tag as a container instead of using` div` tag as a container. The primary issue with heavy use of `<div>` tags is that they lack semantic meaning. Writing Semantic HTML gives your markup meaning to web browsers and screen readers, can help with SEO, makes it easier to debug code, and more. According to the W3C: “The div element has no special meaning at all…